mylyn.docs/wikitext/core/org.eclipse.mylyn.wikitext.ant.tests/META-INF/MANIFEST.MF

```diff
@@ -5,7 +5,7 @@ Bundle-Version: 4.2.0.qualifier
 Bundle-Name: Mylyn WikiText Ant Tasks Tests
 Bundle-Vendor: Eclipse Mylyn
 Fragment-Host: org.eclipse.mylyn.wikitext.ant
-Bundle-RequiredExecutionEnvironment: JavaSE-11
+Bundle-RequiredExecutionEnvironment: JavaSE-17
 Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=11))"

```

Comment:
You have to update the osgi.ee require capability to Java 17 too.

Reply:
Fixed